R/print.R

```diff
@@ -38,6 +38,7 @@ model_descs <- tibble::tribble(
   ~cls,                   ~desc,
   "auto_ml",              "Automatic Machine Learning",
   "bag_mars",             "Bagged MARS",
+  "bag_mlp",              "Bagged Neural Networks",
```

Comment:
```suggestion
  "bag_mlp",              "Bagged Neural Network",
```

...I think? Following the lead of "Bagged Decision Tree" here.